### PR TITLE
[branch] old-seasonals: fix all syntax and style errors

### PR DESCRIPTION
**Pull request by codelegend**
Style based on current gulpfile.

Two issues to be resolved:
- `randomSeasonalTeam`: both **Seasoning's Greetings** and **Thankless Thanksgiving** use the same name. Scripts at lines 9 and 546
- `randomSeasonalFFTeam`: both **Fools Festival** and **Fabulous February** use it. Scripts at lines 173 and 600

### CODE REVIEW

**Comment by codelegend** [comment]
@Zarel , I think you should disable travis builds for branches without a .travis.yml (ie build only if .travis.yml exists)

File: formats.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  *
  * THIS IS A BACKUP FILE.
  */
- exports.BattleFormats = {
+exports.Formats = [
```
Comment (RhythmsAreFromVibrations) [inline formats.js]: Why...

Comment (codelegend) [inline formats.js]: for why `Formats`: look at `config/formats.js`
for why `[`: it is really an array, not an object.

**Comment by Zarel** [comment]
@Joimer?

File: formats.js (hunk 1 above)
Comment (Joimer) [inline formats.js]: It's a backup, it tries to keep the same code as the server. I mean, it's not plug and play code, but it should follow the same style.

The other changes following gulpfile are good, thanks!

**Comment by Zarel** [comment]
Also what's wrong with Travis CI here?

**Comment by Zarel** [comment]
Oh, this branch is before we actually had any tests at all.

**Comment by Joimer** [comment]
Could not find .travis.yml, using standard configuration. And failed with it. It needs a config file.

Damn, phone is obnoxious for checking that.

**Comment by codelegend** [comment]
@Joimer , what about those two issues i mentioned?

**Comment by Joimer** [comment]
Well, it's just a back-up file, doesn't really matter, but I'd like to keep style, fix if you want.

**Comment by codelegend** [comment]
ok, fixed in #1608

**Pull request merged**